src/lnp/application/payment/types.rs

```diff
@@ -262,13 +263,21 @@ impl DumbDefault for TempChannelId {
     }
 }
 
+#[derive(Wrapper, Clone, Debug, From, PartialEq, Eq)]
+pub struct RGBColor([u8; 3]);
+
+impl Strategy for RGBColor {
+    type Strategy = strategies::AsWrapped;
+}
+
+#[derive(Wrapper, Clone, Debug, From, PartialEq, Eq)]
+pub struct Alias([u8; 3]);
```

Comment:
Btw, where this type is used?

Reply:
Here

https://github.com/LNP-BP/rust-lnpbp/blob/f1b4cf05c017928684eb8159a239a132abb81960/src/lnp/application/message.rs#L725

Reply:
Well, it must be `[u8; 32]`, not `[u8; 3]`: https://github.com/lightningnetwork/lightning-rfc/blob/master/07-routing-gossip.md#the-node_announcement-message